### ISSUE TITLE
pending issues on cuDF

### ISSUE THREAD

**Issue by jangorecki** [h2oai/db-benchmark#148]
This issue is to list other pending issues around `cudf` and provide current status of each. The status may not be clear from issues directly, because we had to roll back some of the improvements. I will keep updating this first post, rather adding new posts here. Comments related to any particular issue should go into specific linked issues.

- [ ] using multiple GPUs #116: need to use `dask_cudf`
- [ ] spil vmem to main memory #129: we had to rollback this because `cudf` is not handling out of memory error, rendering gpu in corrupted state, it can reliably work only via `dask_cudf`
- [ ] use `dask_cudf`: blocked by lack of documentation rapidsai/cudf#2277
- [ ] run 1e8 and 1e9 data cases #97: spil to main mem could possibly allow to complete 1e8 size, for 1e9 we will need spil to disk memory, for both dask is required, so it is blocked on `dask_cudf` as well
- [x] check if cuDF join materializes answer #155
- [ ] cudf with dask does not yet have median implemented #216
- [ ] cudf via dask fail with groupby q10 #217
- [ ] cudf dask join on categorical fails #218

**Comment by jangorecki** [comment]
All the long standing issues has been recently resolved by updated documentation on usage of dask cudf. This issue can be now closed. There are multiple new issue created but their status is very clear and we don't have to maintain master issue anymore.
